tony/README.md

```diff
@@ -0,0 +1,49 @@
+# TonY - TensorFlow over YARN
+
+This [initialization action](https://cloud.google.com/dataproc/init-actions) installs the latest version of [TonY](https://github.com/linkedin/TonY)
+on a master node within a [Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster.
+
+## Using this initialization action
+
+You can use this initialization action to create a new Dataproc cluster with TonY installed:
+
+1. Use the `gcloud` command to create a new cluster with this initialization action.  The following command will create a new cluster named `<CLUSTER_NAME>`.
+
+    ```bash
+    gcloud dataproc clusters create <CLUSTER_NAME> \
+      --initialization-actions gs://dataproc-initialization-actions/tony/tony.sh
+    ```
+
+    You can also pass specific metadata:
+    
+    ```bash
+    gcloud dataproc clusters create <CLUSTER_NAME> \
+      --initialization-actions gs://dataproc-initialization-actions/tony/tony.sh \
+      --metadata name1=value1,name2=value2... 
+    ```
+    
+    Supported metadata parameters:
+    
+     - num_workers: 
+     - worker_memory
+     - ps_memory
+        
+    These parameters are defined here: TonY [configurations](https://github.com/linkedin/TonY/wiki/TonY-Configurations)
```

Comment:
The short name you've used above (`num_workers`, `worker_memory`, `ps_memory`) don't exist on the TonY configurations page but only in the `tony.sh` script. Perhaps you can tell users to consult the tonY configurations page if they need to configure or adjust other settings not supported by the `tony.sh` script.